src/Playwright.Tests/BrowserContextExposeFunctionTests.cs

```diff
@@ -129,5 +129,27 @@ public async Task ExposeBindingHandleLikeInDocumentation()
             await page.ClickAsync("div");
             Assert.AreEqual("Click me", await result.Task);
         }
+
+        [Test, Timeout(TestConstants.DefaultTestTimeout)]
+        public async Task ExposeBindingHandleLikeInDocumentationButInAWeirdWay()
+        {
+            var result = new TaskCompletionSource<string>();
+            var page = await Context.NewPageAsync();
+            System.Action<BindingSource, IJSHandle> act = async (_, t) =>
+            {
+                result.TrySetResult(await t.AsElement().TextContentAsync());
+            };
+
+            await Context.ExposeBindingAsync("clicked", act, new() { Handle = true });
+
+            await page.SetContentAsync("<script>\n" +
+             "  document.addEventListener('click', event => window.clicked(event.target));\n" +
+             "</script>\n" +
+             "<div>Click me</div>\n" +
+             "<div>Or click me</div>\n");
```

Comment:
This will look prettier with `@""`

Reply:
But the rest of the file has this syntax, so I'm keeping it. At some point, we need to standardize on using `@""` and fix *all* of these.